articles/bottom-navigation-bar-in-android/index.md

```diff
@@ -0,0 +1,290 @@
+
+* Three to five views.
+* Views that need direct access.
+
+Popular mobile applications that use this navigation component are Instagram, Twitter, Youtube, etc.
+
+![Youtube](/engineering-education/bottom-navigation-bar-in-android/youtube.jpg)
+
+![Instagram](/engineering-education/bottom-navigation-bar-in-android/instagram.jpg)
+
+
+![Twitter](/engineering-education/bottom-navigation-bar-in-android/twitter.jpg)
+
+### Structure of a Bottom Navigation Bar
+A Bottom Navigation bar is a container which contains the following:
+* Inactive icon
+* Inactive text label (optional)
+* Active icon
+* Active text label
+
+
+![Structure of a Bottom Navigation Bar](/engineering-education/bottom-navigation-bar-in-android/structure.jpg)
+
+### Structure of a Navigation Component.
+A  Navigation component consists of the following:
+* Navigation graph
+* NavHost
+* NavController
+
+### Advantages of a Bottom Navigation View using a Navigation Component.
+* It makes applications more user friendly.
+* It eases the development processes of an application.
+* It enables implementing and handling Deep Linking.
+* It allows the handling of fragment transactions.
+* It Offers `ViewModel` support
+* It offers animations and transitions support.
+
+
+### Disadvantages of a Bottom Navigation View using a Navigation Component.
+* Applications with less than three destinations cannot use it.
+* Limited to only Mobiles and Tablets.
+* It is challenging to implement a `TabLayout`
+
+
+
+### Terminologies
+* [Bottom Navigation Bar](https://material.io/develop/android/components/bottom-navigation) - a navigation component that enables users to explore and change to different applications' views.
+* [Dependency](https://developer.android.com/studio/build/dependencies) - a statement SDK that allows us to add an external library into our projects.
+* [Fragment](https://developer.android.com/reference/android/app/Fragment) - a fragment is a sub-activity which enables more modular activity design.
+* [Navigation Component](https://developer.android.com/guide/navigation/navigation-getting-started) - a resource file that contains information used for navigation.
+* [Navigation Graph](https://developer.android.com/guide/navigation/navigation-getting-started#create-nav-graph) - a resource file that contains information used for navigation.
+* [NavHost](https://developer.android.com/reference/androidx/navigation/NavHost) - an empty container used to display destinations contained in the navigation graph.
+* [NavController](https://developer.android.com/reference/androidx/navigation/NavController) - an object within the `NavHost` which manages navigation.
+
+
+### Prerequisites
+* [Android Studio](https://developer.android.com/studio) installed.
+
+#### Step 1 – Create a new Project.
```

Comment:
You could have a paragraph describing this process of starting a new AS project instead of having many images. Plus, it is a straight forward process that can be summarised using arrows.